clock-mcu/src/firmware/clock.h

```diff
@@ -74,12 +74,15 @@ void handleTime() {
 #endif 	// RTC_ENABLED
 
 	// interval for flashing time display separator colon
-	if (millis() - prevSeparatorTime >= separatorInterval) {
-		flipTimeDisplaySeparator();
+  if (millis() - prevSeparatorTime >= separatorInterval) {
+		if (mcuConfig.blink == 2 || (mcuConfig.blink == 1 && bDay)) 
```

Comment:
Seen that in the code, and since you already have `bDay` that's supposed to be true when it's daytime, I guessed that in the future, the `bDay` could be set from the ambient sensor without impacting this code.

Reply:
I doubt you'd want the ambient sensor setting day/night on the clock, since it would think its night if you took a nap during the day. I would want the clock to dim and possibly not blink(personally the blinking doesnt bother me) but I wouldn't want the day night indicator to say night at noon if I am taking a nap...

Reply:
Ok, that make sense and it seems like a feature request.
So we'd add a new option 2 for "don't blink only if light is low" and 3 for "always blink", right ?

How far it is to have a working ambient sensor ?

Reply:
> I doubt you'd want the ambient sensor setting day/night on the clock, since it would think its night if you took a nap during the day. 

Agreed.
Let's keep `bDay` as is. Because its main use is to control the day-night indicator.

Feel free to add new variables to track things.